src/components/Courses/components/CourseCard/CourseCard.jsx

```diff
@@ -47,6 +52,11 @@ const CourseCard = ({ course }) => {
 						<Link to={`/courses/${id}`}>
 							<Button name={buttonNames.showCourseButton} />
 						</Link>
+						<Button
+							name={buttonNames.deleteCourseButton}
```

Comment:
Button does not render icon, please refer to the [doc](https://d17btkcdsmqrmh.cloudfront.net/react-fundamentals/docs/module-3/home-task/components#coursecard-component).
<img width="444" alt="Screenshot 2024-02-28 at 19 50 04" src="https://github.com/VladimirSadovoi/ReactCourses/assets/157737324/ea20e655-b019-4d83-8c6f-26abfb7ed756">